spring-cloud-stream-binder-kinesis/src/test/java/org/springframework/cloud/stream/binder/kinesis/LocalKinesisResource.java

```diff
@@ -53,7 +49,6 @@ protected void obtainResource() throws Exception {
 		System.setProperty(SDKGlobalConfiguration.AWS_CBOR_DISABLE_SYSTEM_PROPERTY, "true");
 
 		this.resource = AmazonKinesisAsyncClientBuilder.standard()
-				.withCredentials(mock(AWSCredentialsProvider.class))
```

Comment:
I don't understand how does this work for you.
I have:
```
org.junit.AssumptionViolatedException: Skipping test due to KINESIS not being available com.amazonaws.SdkClientException: Unable to load AWS credentials from any provider in the chain
```
and it is still too far from the real connection to the `kinesalite`.

When I make it like this:
```
.withCredentials(new AWSStaticCredentialsProvider(new BasicAWSCredentials("", "")))
```
The connection to my local `kinesalite` in the Nod.js starts to work.
And voila!, I have some tests passed, but most of them don't work anyway. 😢